Automattic/jetpack
# RNA: Add Jetpack footer

#### Changes proposed in this Pull Request:
Added two components,
1. A8C SVG Title: 'An Automattic Airline'
2. Jetpack Admin footer, uses the SVG title above and a Jetpack branding
3. Added `plugin:@wordpress/eslint-plugin/i18n` to check i18n text domain.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Go to `projects/js-packages/components` and run `pnpm run test`
- Ensure unit tests pass


- Checkout #20252  and follow test instructions
- Ensure components in footer are rendered properly

- Ensure eslint fails if i18n text domain is set to strings other than `jetpack`

![image](https://user-images.githubusercontent.com/1425433/124697469-e485c000-df3a-11eb-8616-43961361b55d.png)
